app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt

```diff
@@ -27,33 +31,38 @@ import java.util.Locale
  * such as AboutLibraries (https://github.com/mikepenz/AboutLibraries)
  * but we considered the risk of introducing such third-party dependency
  * to Fenix too high. Therefore, we use Google's gradle plugin to
- * extract the dependencies and their licenses, and this activity
+ * extract the dependencies and their licenses, and this fragment
  * to show the extracted licenses to the end-user.
  */
-class AboutLibrariesActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+class AboutLibrariesFragment : Fragment() {
+    override fun onCreateView(
```

Comment:
`onViewCreated` is called immediately after and lets us just use the fragment constructor to set up the layout.
```suggestion
    override fun onViewCreated(
```